pyV2DL3/script/v2dl3_for_vegas.py

```diff
@@ -4,29 +4,22 @@
 import click
 
 from pyV2DL3.generateObsHduIndex import create_obs_hdu_index_file
-from pyV2DL3.vegas.EventClass import EventClass
+from pyV2DL3.vegas.EffectiveAreaFile import EffectiveAreaFile
 
 
-# In event_class_mode, The user can provide multiple EAs per tag to define the event classes for that group.
-def runlist_to_file_pair(rl_dict, event_class_mode=False):
+def runlist_to_file_pair(rl_dict):
     eas = rl_dict["EA"]
     st5s = rl_dict["RUNLIST"]
     file_pair = []
-    # If using multiple EAs for event classes
-    if event_class_mode:
-        for k in st5s.keys():
-            event_classes = []
-            for ea in eas[k]:
-                event_classes.append(EventClass(ea))
-            if len(event_classes) == 0:
-                raise Exception("No EA filenames defined for runlist tag: " + k)
-            for f in st5s[k]:
-                file_pair.append((f, event_classes))
-    else:
-        for k in st5s.keys():
-            ea = eas[k][0]
-            for f in st5s[k]:
-                file_pair.append((f, ea))
+    for k in st5s.keys():
+        ea_files = []
```

Comment:
Isn't this a classical case for list comprehension? Would save two lines and make it more pythonic.
(but not super important)

Reply:
Good idea. My next update cleans and moves this function to match the other ones at the bottom of the file. I will update it there.